### ISSUE TITLE
Auto tamper

### ISSUE THREAD

**Issue by cankaptan1976** [sqlmapproject/sqlmap#1236]
Hi 

How can we try all tamper scripts one by one is there any parameter --auto-tamper like that ?

Thanks in advance.

**Comment by stamparm** [comment]
Imagine one such switch.

There are 42 tamper scripts this moment
There are >5 SQLI techniques supported by sqlmap
There are 5 level values (each level value multiplies number of required requests)
There are 3 risk values (each risk value multiplies number of required requests)
There are indefinite number of target parameters

To do some math. There are tens of thousands requests currently being performed per each parameter when you use high level and risk values. Multiply that by number of tamper scripts. Multiply that per available target parameter values.

In short, answer is no. It won't be implemented. Until someone gives a valid step by step use case (other than "it would be so cool to have it")

**Comment by seyedhojjathosseini** [comment]
https://github.com/seyedhojjathosseini/sqlmap-AutoTamper

> 
> 
> Hi
> 
> How can we try all tamper scripts one by one is there any parameter --auto-tamper like that ?
> 
> Thanks in advance.

https://github.com/seyedhojjathosseini/sqlmap-AutoTamper

**Comment by cancanun** [comment]
stamparm is just a karen,  search it on github you will find it.
